FreeRTOS/FreeRTOS
# Disable new handler installation check for XMC1000

Description
-----------
FreeRTOS/FreeRTOS-Kernel#832 introduced a new runtime feature in the Cortex M ports to verify proper installation of the FreeRTOS handlers into the interrupt vector table.  This new feature is enabled by default but is not compatible with XMC1000.  This PR adds a configuration item to FreeRTOS.h in the XMC1000 demos to disable the new feature.

Without this PR, the new handler-installation check triggers configASSERT().  After this PR, no assert is triggered.

Test Steps
-----------
*Appreciate if the team could check demo execution on real hardware.  (I don't have the hardware.)*

Run any of the XMC1000 demos (Demo\CORTEX_M0_Infineon_XMC1000_IAR_Keil_GCC).  Without this PR, the handler installation check triggers configASSERT().  After this PR, no assert is triggered.

Checklist:
----------


- [ ] I have tested my changes. No regression in existing tests. (Need help please)
- [X] I have modified and/or added unit-tests to cover the code changes in this Pull Request. (N/A)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
